data/user-personas/contributors/code-contributor.yaml

```diff
@@ -5,15 +5,15 @@ index: 0
 foundational:
   - label: "Read the Developer Guide"
     icon: fa-book
-    url: "/docs/community/devel/"
+    url: "https://git.k8s.io/community/contributors/devel"
 intermediate:
   - label: "Learn about the Kubernetes Enhancement Proposal (KEP) process"
     icon: fa-upload
-    url: "/docs/community/keps/"
+    url: "https://git.k8s.io/community/keps/"
   - label: "Understand the API conventions"
     icon: fa-map-o
-    url: "https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md"
+    url: "https://git.k8s.io/community/contributors/devel/api-conventions.md"
 advanced:
   - label: "Learn about the release process"
     icon: fa-code-fork
-    url: "https://github.com/kubernetes/release/blob/master/README.md"
+    url: "https://git.k8s.io/release/README.md"
```

Comment:
@tengqm are these files imported too? Or is it just files from the community repo?

Reply:
@shubheksha Check this: https://github.com/kubernetes/website/tree/master/update-imported-docs

The difference between direct linking and importing files is that the later is meant to be browsed as web pages while the former is meant for contributors to check the markdown source files.

Reply:
As for files to be imported, they can be customized using this file: https://github.com/kubernetes/website/blob/master/update-imported-docs/community.yml